dist/components/top-bar/top-bar.html

```diff
@@ -18,74 +18,34 @@
 registration/signin callout is shown or whether the user avatar is shown.
 -->
 <dom-module id="top-bar">
-	<style>
-		[unresolved] {
-			visibility: hidden;
-		}
-
-		:host {
-			display: block;
-			background-color: #fff;
-			border-bottom: 1px solid #ccc;
-			height: 46px;
-			position: fixed;
-			width: 100%;
-			z-index: 9999999;
-		}
-
-		.logo {
-			fill: #092344;
-			width: 82px;
-		}
-
-		.user-status {
-			float: right;
-		}
-
-		.user-status figure {
-			margin-top: 3px;
-			width: 36px;
-			height: 36px;
-			border-radius: 50%;
-			overflow: hidden;
-		}
+	<template>
+		<style>
+			[unresolved] {
+				visibility: hidden;
+			}
 
-		.user-status figure img {
-			width: 100%;
-			height: auto;
-		}
+			:host {
+				display: block;
```

Comment:
Make sure these styles are alphabetically sorted. I think this is the only declaration I saw that's out of order.